src/content/docs/es/guides/cms/hashnode.mdx

```diff
@@ -0,0 +1,353 @@
+---
+title: Hashnode & Astro
+description: Agrega contenido a tu proyecto de Astro usando Hashnode como un CMS
+type: cms
+stub: false
+service: Hashnode
+i18nReady: true
+---
+
+import { FileTree } from '@astrojs/starlight/components';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import { Steps } from '@astrojs/starlight/components';
+
+[Hashnode](https://hashnode.com/) es un CMS alojado que te permite crear un blog o publicación.
+
+## Integración con Astro
+La [API Pública de hashnode](https://apidocs.hashnode.com/) es una API GraphQL que te permite interactuar con Hashnode. Esta guía utiliza [`graphql-request`](https://github.com/jasonkuhrt/graphql-request), un cliente GraphQL minimalista que funciona bien con Astro, para traer tus datos de Hashnode a tu proyecto de Astro.
+
+### Requisitos Previos
+Para comenzar necesitarás tener lo siguiente:
+
+1. **Un proyecto de Astro** - Si aún no tienes un proyecto de Astro, nuestra [Guía de Instalación](/es/install/auto/)te ayudará a empezar en poco tiempo.
+
+2. **Un sitio en Hashnode** - Puedes crear un sitio personal gratuito visitando [Hashnode](https://hashnode.com/).
+
+### Instalando dependencias
+
+Instala el paquete `graphql-request` usando el gestor de paquetes de tu elección:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install graphql-request
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add graphql-request
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add graphql-request
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+## Haciendo un blog con Astro y Hashnode
+
+Esta guía usa [`graphql-request`](https://github.com/jasonkuhrt/graphql-request), un cliente GraphQL minimalista que funciona bien con Astro, para traer tus datos de Hashnode a tu proyecto de Astro.
+
+### Requisitos Previos
```

Comment:
```suggestion
### Prerrequisitos
```

Reply:
Thanks for the feedback @dreyfus92!! 
You're right!! i should change that!